esrally/driver/runner.py

```diff
@@ -23,7 +23,7 @@
 import types
 from collections import Counter, OrderedDict
 from copy import deepcopy
-
+from elasticsearch.client import _make_path
```

Comment:
Can we please avoid top-level imports of Elasticsearch (logging issues)? Also, I am not certain why we need this? Can you please clarify?

Reply:
see below. Moving to local import.